frontend/dashboard/components/RepoList/ActionLinks.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, DropdownMenu } from '@digdir/design-system-react';
+import React, { useRef, useState } from 'react';
+import classes from './RepoList.module.css';
```

Comment:
Should this ActionLinks component import css classes from its own css module?

Reply:
Something like this? 😊 

<img width="168" alt="image" src="https://github.com/Altinn/altinn-studio/assets/148075168/3b693585-9a1b-45cf-beaa-5b89a7a1cc32">